### PR TITLE
ref(rename logo): replace icon with logo component

### PR DESCRIPTION
**Pull request by doralchan**
- Added LogoSentry component that has props to use wordmark or glyph

Screen Shot 2021-05-03 at 2 09 55 PM
Screen Shot 2021-05-03 at 2 09 04 PM

### CODE REVIEW

File: static/app/components/logoSentry.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+type Props = {
+  showWordmark: boolean;
+  height: string;
+};
+
+const LogoSentry = ({showWordmark, height}: Props) => {
```
Comment (priscilawebdev) [inline static/app/components/logoSentry.tsx:8]:
```suggestion
function LogoSentry({showWordmark = true, height = '32px'}: Props) {
```

File: static/app/components/logoSentry.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 44" height={height}>
+      {showWordmark ? wordmarkSvg : glyphSvg}
+    </svg>
+  );
+};
+
+LogoSentry.defaultProps = {
+  showWordmark: true,
+  height: '32px',
+};
```
Comment (priscilawebdev) [inline static/app/components/logoSentry.tsx:27]:
```suggestion

```

File: static/app/components/logoSentry.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  );
+
+  const glyphSvg = (
+    <path d="M29,2.26a4.67,4.67,0,0,0-8,0L14.42,13.53A32.21,32.21,0,0,1,32.17,40.19H27.55A27.68,27.68,0,0,0,12.09,17.47L6,28a15.92,15.92,0,0,1,9.23,12.17H4.62A.76.76,0,0,1,4,39.06l2.94-5a10.74,10.74,0,0,0-3.36-1.9l-2.91,5a4.54,4.54,0,0,0,1.69,6.24A4.66,4.66,0,0,0,4.62,44H19.15a19.4,19.4,0,0,0-8-17.31l2.31-4A23.87,23.87,0,0,1,23.76,44H36.07a35.88,35.88,0,0,0-16.41-31.8l4.67-8a.77.77,0,0,1,1.05-.27c.53.29,20.29,34.77,20.66,35.17a.76.76,0,0,1-.68,1.13H40.6q.09,1.91,0,3.81h4.78A4.59,4.59,0,0,0,50,39.43a4.49,4.49,0,0,0-.62-2.28Z" />
+  );
+
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 44" height={height}>
```
Comment (priscilawebdev) [inline static/app/components/logoSentry.tsx:18]: should we use the `SvgIcon` component here? maybe we should rename it to Svg 🤔

**Review by markstory — APPROVED** [review]
Looks good with Priscila's changes.

File: static/app/components/logoSentry.tsx (hunk 3 above)
Comment (doralchan) [reply static/app/components/logoSentry.tsx:18]: The logo wordmark and glyph will not be in the same sized viewbox or need to inherit colors as readily as icons. There's a stricter usage for this so I think it should be fine to leave it out.

**Pull request merged**